README.md

```diff
@@ -132,12 +132,12 @@ We can divide the problem in 3 big steps.
 * **AI Runner**. This will take all the user provided functions and the current game state, and execute every function.
   * This will take care of catch errors on the functions, and stop non-responding functions to hang the window.
 * **Game Core**. This will take the responses that the AI Runners sends, and apply the game logic on them.
-  * Kill killed players.
+  * Handle killed players.
   * Move and turn players.
   * Collect and count coins.
   * Generate new coins if necessary.
-  * Set the paralized turns to players that shooted.
-  * Count if too many inactive turns had passed.
+  * Set the paralized turns to players that shot.
```

Comment:
I wasn't sure what was meant by paralized turns here, so the wording may need slight changes.  Let me know if you have a preference and I can change it if you'd like.